include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce_then_scan.h

```diff
@@ -702,6 +728,16 @@ struct __parallel_reduce_then_scan_scan_submitter<
     _InitType __init;
 };
 
+// reduce_then_scan requires subgroup size of 32, and performs well only on devices with fast coordinated subgroup
+// operations.  We do not want to run this can on CPU targets, as they are not performant with this algorithm.
+template <typename _ExecutionPolicy>
+bool
+__is_best_alg_reduce_then_scan(_ExecutionPolicy&& __exec)
```

Comment:
Not a big deal but since we don't forward the forwarding reference, should we just take it as `const _ExecutionPolicy& __exec`?

Reply:
sure, done